docs/code/code-repo-management.asciidoc

```diff
@@ -46,9 +64,26 @@ xpack.code.security.gitProtocolWhitelist: [ "https" ]
 ----
 
 [float]
-==== Clone repo with SSH key
-If your repo clone requires an SSH key for authentication, put the SSH key in `data/code/credentials/` under the {kib} folder.
+==== Clone private repo with SSH key
+Clones of private repos require an SSH key for authentication. The username associated with your host must have write access to the repository you want to clone.
+
+The following section provides links for generating your ssh key through GitHub. If you already have an SSH key added through your host, skip to step 4.
+
+1. https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key[Generate an ssh key].
+
+2. https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent[Add the ssh key to your ssh-agent.]
 
+3. https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account[Add the ssh key to your host].
+
+4. Copy the ssh key into `data/code/credentials/` under the {kib} folder.
+
+You can now copy private Git repositories into Code.
+
+[float]
+==== Limitations
+Consider the following limitations before cloning repositories:
 
+- Only Git is supported. Other version control systems such as Mercurial and SVN are not supported.
```

Comment:
Suggest separating Mercurial & SVN phrase with commas:  Other version control systems, such as Mercurial and SVN, ....